decidim-verifications/config/locales/en.yml

```diff
@@ -79,8 +79,8 @@ en:
     events:
       verifications:
         verify_with_managed_user:
-          email_intro: The participant <a href="%{resource_path}">%{resource_title}</a> has tried to verify themself with the data of the managed participant <a href="%{managed_user_path}">%{managed_user_name}</a>.
-          email_outro: Check the <a href="%{conflicts_path}">Verifications's conflicts list</a> and contact the participant to verify their details and solve the issue.
+          email_intro: The participant <a href="%{resource_url}">%{resource_title}</a> has tried to verify themself with the data of the managed participant <a href="%{managed_user_url}">%{managed_user_name}</a>.
```

Comment:
When implementing #9463, I have noticed conflicts are also possible between the registered users, so I was just thinking should we replace the "managed participant" in the string with "other participant"?

Reply:
Sure, let's handle it in other PR. My comment was only regarding the string (which I found confusing before).